cmd/fyne_demo/main.go

```diff
@@ -253,10 +253,10 @@ func makeNav(setTutorial func(tutorial tutorials.Tutorial), loadPrevious bool) f
 
 	themes := container.NewGridWithColumns(2,
 		widget.NewButton("Dark", func() {
-			a.Settings().SetTheme(&forcedVariant{Theme: theme.DefaultTheme(), variant: theme.VariantDark})
+			//a.Settings().SetTheme(&forcedVariant{Theme: theme.DefaultTheme(), variant: theme.VariantDark})
```

Comment:
I've seen these changes commited many times in your PRs. Is there any specific reason for ding this? Just curious :)

Reply:
Ah, I see. Your `go run` command is incorrect. Do `go run ./cmd/fyne_demo` (or `go run .` if you are in the same folder) and it will all work fine.